datalake/src/test/java/com/sequenceiq/datalake/service/sdx/start/SdxStartServiceTest.java

```diff
@@ -43,6 +44,8 @@
 
     private static final Long CLUSTER_ID = 1L;
 
+    private static final String ENV_CRN = "env_crn";
```

Comment:
**suggestion:** Consider a well-formed CRN here, I've seen tests fail when we use the CRN constructor internally and feed it a String that isn't actually a valid CRN.

Here's one from one of my environments:
```
crn:cdp:environments:us-west-1:9d74eee4-1cad-45d7-b645-7ccf9edbb73d:environment:4044a133-e941-48d4-82c4-ce1e231160d9
```